[export] Log export time.

zhxchen17 opened the pull request:
Summary: as title. we are logging the time to complete one export session.

Test Plan: CI

Differential Revision: D53737766

angelayi commented on torch/export/_trace.py:
```diff
@@ -742,6 +747,8 @@ def forward(self, *args, **kwargs):
             example_inputs=(args, kwargs),
             constants=ep_non_strict.constants,
         )
+        ep._flags = flags
```
what's the flags for?

zhxchen17 replied:
ok nvm it doesn't quite work lol. I use a global instead.

zhxchen17 committed:
[export] Log export time. (#119960)

Summary:

as title. we are logging the time to complete one export session.

Test Plan: CI

Reviewed By: angelayi

Differential Revision: D53737766

huydhn commented:
@pytorchbot merge -f 'All failures are pre-existing broken trunk failures'
